src/hash.jl

```diff
@@ -99,6 +99,7 @@ begin
     hash_idx += 1
   end
 end
+__init__()
```

Comment:
v"0.3-" is usually a better version to specify. but, yes, that's the pattern i've used in Gtk for this same purpose